src/app/public/modules/wait/wait.component.ts

```diff
@@ -41,17 +40,48 @@ export class SkyWaitComponent {
 
     this._isFullPage = value;
   }
-
   public get isFullPage() {
     return this._isFullPage;
   }
 
+  @Input()
+  public set ariaLabel(value: string) {
+    this._ariaLabel = value;
+  }
+  public get ariaLabel(): string {
+    if (this._ariaLabel) {
+      return this._ariaLabel;
+    }
+
+    const type = this.isFullPage ? '_page' : '';
+    const blocking = this.isNonBlocking ? '' : '_blocking';
+    const resourceKey = 'sky_wait' + type + blocking + '_aria_alt_text';
+    return this.getString(resourceKey);
```

Comment:
Since `ariaLabel` is a new input, we wouldn't be making any breaking changes to an existing API to implement `SkyAppResources`. I'd figure out a solution that allows you to set `ariaLabel`, but use `SkyAppResources` for the default value.